cores/esp32/esp32-hal-uart.c

```diff
@@ -69,23 +76,48 @@ static uart_t _uart_bus_array[3] = {
 
 static void uart_on_apb_change(void * arg, apb_change_ev_t ev_type, uint32_t old_apb, uint32_t new_apb);
 
-static void IRAM_ATTR _uart_isr(void *arg)
+static uart_interrupt_t *_uart_interrupt_array[3] = {NULL, NULL, NULL};
+
+static void IRAM_ATTR _uart_isr()
```

Comment:
here `_uart_isr` lost the `void *arg` argument. Is that intentional?

Reply:
No, in my complete auto pilot I removed the parameter after setting NULL as an argument in esp_intr_alloc.
Fixed